data_engineering/derive_pipeline_runner.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+
+from utils import pipeline_io
+
+from data_engineering.derive.derive import get_raw_values
+
+
+def run():
+    train_df = pipeline_io.read_train_file()
+    print(train_df.head())
+
+    raw_values_df = get_raw_values(train_df)
+
+    pipeline_io.save_raw_values_file(raw_values_df)
+
+
+if __name__ == "__main__":
+    run()
```

Comment:
I would prefer to only have one pipeline runner, and refactoring this into a function, added to the pipeline_runner, what do you think?

Reply:
but i think these two different pipelines are doing different jobs? one is to get raw values from ratio and the other is to clean the ratio directly. Eventually we will see which one works better on our model so we can pick a final DE pipeline but for now I think we can leave it first?

Reply:
hmmm I thought we will use this one directly... so we still have hope to the ratio data? hahaha

Reply:
Yeah I would prefer one pipeline too. I think with one pipeline we can create 2 outputs at the same time — one is reverse engineered, another one is original version.